n3fit/runcards/examples/nnpdf40-like.yml

```diff
@@ -0,0 +1,167 @@
+- {dataset: ATLAS_TTBAR_13TEV_TOT_X-SEC, frac: 0.75, variant: legacy}
+- {dataset: ATLAS_TTBAR_8TEV_LJ_DIF_YT-NORM, frac: 0.75, variant: legacy}
+- {dataset: ATLAS_TTBAR_8TEV_LJ_DIF_YTTBAR-NORM, frac: 0.75, variant: legacy}
+- {dataset: ATLAS_TTBAR_8TEV_2L_DIF_YTTBAR-NORM, frac: 0.75, variant: legacy}
+- {dataset: ATLAS_1JET_8TEV_R06_PTY, frac: 0.75, variant: legacy_decorrelated}
+- {dataset: ATLAS_2JET_7TEV_R06_M12Y, frac: 0.75, variant: legacy}
+- {dataset: ATLAS_PH_13TEV_XSEC, frac: 0.75, cfac: [EWK], variant: legacy}
+- {dataset: ATLAS_SINGLETOP_7TEV_TCHANNEL-XSEC, frac: 0.75, variant: legacy}
+- {dataset: ATLAS_SINGLETOP_13TEV_TCHANNEL-XSEC, frac: 0.75, variant: legacy}
+- {dataset: ATLAS_SINGLETOP_7TEV_T-Y-NORM, frac: 0.75, variant: legacy}
+- {dataset: ATLAS_SINGLETOP_7TEV_TBAR-Y-NORM, frac: 0.75, variant: legacy}
+- {dataset: ATLAS_SINGLETOP_8TEV_T-RAP-NORM, frac: 0.75, variant: legacy}
+- {dataset: ATLAS_SINGLETOP_8TEV_TBAR-RAP-NORM, frac: 0.75, variant: legacy}
+- {dataset: CMS_WPWM_7TEV_ELECTRON_ASY, frac: 0.75}
+- {dataset: CMS_WPWM_7TEV_MUON_ASY, frac: 0.75, variant: legacy}
+- {dataset: CMS_Z0_7TEV_DIMUON_2D, frac: 0.75}
+- {dataset: CMS_WPWM_8TEV_MUON_Y, frac: 0.75, variant: legacy}
+- {dataset: CMS_Z0J_8TEV_PT-Y, frac: 0.75, cfac: [NRM], variant: legacy_10}
+- {dataset: CMS_2JET_7TEV_M12Y, frac: 0.75}
+- {dataset: CMS_1JET_8TEV_PTY, frac: 0.75, variant: legacy}
+- {dataset: CMS_TTBAR_7TEV_TOT_X-SEC, frac: 0.75, variant: legacy}
+- {dataset: CMS_TTBAR_8TEV_TOT_X-SEC, frac: 0.75, variant: legacy}
+- {dataset: CMS_TTBAR_13TEV_TOT_X-SEC, frac: 0.75, variant: legacy}
+- {dataset: CMS_TTBAR_8TEV_LJ_DIF_YTTBAR-NORM, frac: 0.75, variant: legacy}
+- {dataset: CMS_TTBAR_5TEV_TOT_X-SEC, frac: 0.75, variant: legacy}
+- {dataset: CMS_TTBAR_8TEV_2L_DIF_MTTBAR-YT-NORM, frac: 0.75, variant: legacy}
+- {dataset: CMS_TTBAR_13TEV_2L_DIF_YT, frac: 0.75, variant: legacy}
+- {dataset: CMS_TTBAR_13TEV_LJ_2016_DIF_YTTBAR, frac: 0.75, variant: legacy}
+- {dataset: CMS_SINGLETOP_7TEV_TCHANNEL-XSEC, frac: 0.75, variant: legacy}
+- {dataset: CMS_SINGLETOP_8TEV_TCHANNEL-XSEC, frac: 0.75, variant: legacy}
+- {dataset: CMS_SINGLETOP_13TEV_TCHANNEL-XSEC, frac: 0.75, variant: legacy}
+- {dataset: LHCB_Z0_7TEV_DIELECTRON_Y, frac: 0.75}
+- {dataset: LHCB_Z0_8TEV_DIELECTRON_Y, frac: 0.75}
+- {dataset: LHCB_WPWM_7TEV_MUON_Y, frac: 0.75, cfac: [NRM]}
+- {dataset: LHCB_Z0_7TEV_MUON_Y, frac: 0.75, cfac: [NRM]}
+- {dataset: LHCB_WPWM_8TEV_MUON_Y, frac: 0.75, cfac: [NRM]}
+- {dataset: LHCB_Z0_8TEV_MUON_Y, frac: 0.75, cfac: [NRM]}
+- {dataset: LHCB_Z0_13TEV_DIMUON-Y, frac: 0.75}
+- {dataset: LHCB_Z0_13TEV_DIELECTRON-Y, frac: 0.75}
+
+############################################################
+datacuts:
+  t0pdfset: 240329-01-rs-nnpdf40like-baseline
+  q2min: 3.49
+  w2min: 12.5
+
+############################################################
+theory:
+  theoryid: 708
+sampling:
+  separate_multiplicative: false
```

Comment:
Just wondering whether we should remove this key by now or set it to true going forward (@andreab1997 ?)

Reply:
If we want to be backward compatible with NNPDF40 we cannot remove the key. Otherwise, if we no longer care about backward compatibility, given that we are approaching 4.1, I agree we should remove the key and just set it to False. Note that the NNPDF40 behaviour is recovered with true not with false.

Reply:
For 1-1 compatibility there is the conda environments provided. I think from 4.0.10 onwards we are happy with slowly moving away from 4.0 compatibility.